index.html

```diff
@@ -87,7 +87,7 @@
 		</div>
 		</nav>
 
-		<header data-aos="fade-up">
+		<header data-aos="fade-up" style="height:100vh">
```

Comment:
why do we need this inline and in `main.scss`

Reply:
It's not gonna overwrite if I do it in `main`, I think its better here since its a single exception to all the other headers on the other pages.

Reply:
I would prefer if give this a class name. use css selector in main.scss to specify this with a comment on why this header is special

Reply:
It's not really special enough to be its own class, it just happens the height should be 100vh and not 130vh for padding. `main.scss` should be used for styles that apply to most tags, many elements. This specific kind of thing should be done inline with bootstrap so I've changed it.

Reply:
Mb, look at the change now. I should use inline bootstrap not custom styles.

Reply:
thanks!